tools/integration_tests.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+# Given a deno executable, this script executes several integration tests with
+# it. The tests are stored in /tests/ and each is specified in a .yaml file
+# where a description, command line, and output are specified.  Optionally an
+# exit code can be specified.
+#
+# Usage: integration_tests.py [path to deno executable]
+import os
+import re
+import sys
+import subprocess
+from util import pattern_match
+
+root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+tests_path = os.path.join(root_path, "tests")
+
+
+def read_test(file_name):
+    with open(file_name, "r") as f:
+        test_file = f.read()
+    lines = test_file.splitlines()
+    test_dict = {}
+    for line in lines:
+        key, value = re.split(r":\s+", line)
+        test_dict[key] = value
+    return test_dict
+
+
+def integration_tests(deno_executable):
+    assert os.path.isfile(deno_executable)
+    tests = sorted([
+        filename for filename in os.listdir(tests_path)
+        if filename.endswith(".test")
+    ])
+    assert len(tests) > 0
+    for test_filename in tests:
+        test_abs = os.path.join(tests_path, test_filename)
+        test = read_test(test_abs)
+        title = test.get("title", test_filename)
+        exit_code = int(test.get("exit_code", 0))
+        deno_args = test.get("deno_args", "").split(" ")
+        output_abs = os.path.join(root_path, test.get("output", ""))
+        with open(output_abs, 'r') as f:
+            expected_out = f.read()
+        cmd = [deno_executable] + deno_args
+        print "Running: %s..." % (title)
+        print "  Command: %s" % (" ".join(cmd))
+        actual_code = 0
+        try:
+            actual_out = subprocess.check_output(cmd, universal_newlines=True)
+        except subprocess.CalledProcessError as e:
+            actual_code = e.returncode
+            actual_out = e.output
+
+        if exit_code != actual_code:
+            print "  🚫  FAIL"
+            print "Expected exit code %d but got %d" % (exit_code, actual_code)
+            print "Output:"
+            print actual_out
+            sys.exit(1)
+
+        if pattern_match(expected_out, actual_out) != True:
+            print "  🚫  FAIL"
```

Comment:
Use the red fail we use in the unit tests. 
https://github.com/denoland/deno/blob/edd12b90a8cc169bb226fcaf273973765844ce0d/js/testing/testing.ts#L61-L71